### DISCUSSION TITLE
Error: NextRouter was not mounted 오류

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#143] [ErrorLog] [answered]
### 🚨 에러

페이지 간 이동을 위해
`import { useRouter } from 'next/router';` 를 등록하였을 때 발생하는 에러
![image](https://github.com/GymHubCommunity/GymHub-FE/assets/52947668/77ca35b8-4b05-4a4b-9fc6-37d561945830)

**Comment by miniposi** [comment, marked as answer]
### 원인
'use client' 를 사용하면 해당 에러가 발생한다고 한다!..

### 해결 방안
`next/router` 대신 `next/navigation`을 사용하자!
`import { useRouter } from 'next/navigation';`
해결 👊💢

**Reply by hanseulhee** [reply]
next/router 에서 사용하던 걸 app router 에서는 next/navigation으로 해야되니 안되는 것도 있고 이건 좀 불편하네요 ...
